src/components/integration/add/IndicationAdd.tsx

```diff
@@ -205,7 +206,7 @@ const IndicationAdd = ({ isEdit, indicationId }: Indication) => {
                 className="w-20 text-right text-base text-violet-400 dark:text-violet-500"
                 htmlFor="apartmentBlock"
               >
-              Bloco AP
+              {t('form-apblock')}
```

Comment:
prefer clear keys, avoid names like this one, set all keys in english `Apartment Block`